vip-cache-manager/vip-cache-manager.php

```diff
@@ -37,7 +37,7 @@ public function __construct() {
 	}
 
 	public function init() {
-		if ( is_super_admin() && isset( $_GET['cm_purge_all'] ) && check_admin_referer( 'manual_purge' ) ) {
+		if ( ! function_exists( 'is_proxied_automattician' ) || ! is_proxied_automattician() && isset( $_GET['cm_purge_all'] ) && check_admin_referer( 'manual_purge' ) ) {
```

Comment:
This probably needs to be a positive check?, i.e. `true === is_proxied_automattician()`

Might be good to to move to a helper method in the class:

```
private function can_purge() {
    return function_exists( 'is_proxied_automattician' ) && is_proxied_automattician();
}
```